### PR TITLE
Logout Dropdown

### PR DESCRIPTION
**Pull request by adeniji19**
Moved the logout option into a dropdown menu.

### CODE REVIEW

File: tigerpath/static/app/css/app-style.css [hunk 1]
```diff
@@ -98,7 +98,7 @@ table tr td:last-child {
 .semester {
   background-color: #eee;
   vertical-align: top;
-  height: calc(50vh - 58px);
+  height: calc(50vh - 60px);
```
Comment (churichard) [inline tigerpath/static/app/css/app-style.css:101]: Why did you change this?

File: tigerpath/static/app/css/app-style.css [hunk 2]
```diff
@@ -141,7 +182,7 @@ thead.semester-header {
   background-color: #464e56!important;
 }
 #display-courses {
-  height: calc(100vh - 127px);
+  height: calc(100vh - 95px);
```
Comment (churichard) [inline tigerpath/static/app/css/app-style.css]: Why did you change this?

File: tigerpath/templates/tigerpath/index.html [hunk 3]
```diff
@@ -152,19 +158,25 @@
...
+    <!-- dropdown menu -->
     <script>
-      function toggler(divId)
-      {
-        var isLogoutHidden = $("#logout-btn").is(":hidden");
-        if ($("#logout-btn").is(":hidden")) {
-          $("#logout-btn").toggle();
-          $("#netid-btn").toggle();
-        }
+      function dropFunction() {
```
Comment (churichard) [inline tigerpath/templates/tigerpath/index.html]: Put this dropdown javascript code in a separate javascript file (in `tigerpath/static/app/js/dropdown.js` for example) and then load the script like this: `<script src="{% static 'app/js/dropdown.js' %}"></script>`

**Review by churichard — CHANGES_REQUESTED** [review]
You should put an arrow next to the netid. Try https://fontawesome.com/icons/caret-down?style=solid

File: tigerpath/static/app/css/app-style.css (hunk 1 above)
Comment (adeniji19) [reply tigerpath/static/app/css/app-style.css:101]: minor fix for different screen sizes

**Review by churichard — APPROVED** [review]
I still think an arrow should be added to the name, but approving this for now.

**Pull request merged**